opencore-amr.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: opencore-amr
+  version: 0.1.6
+  epoch: 0
+  description: Implementation of Adaptive Multi Rate speech codec
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - gcc
+      - glibc-dev
+      - make
+      - pkgconf-dev
+      - wolfi-baselayout
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha512: 8955169954b09d2d5e2190888602c75771b72455290db131ab7f40b587df32ea6a60f205126b09193b90064d0fd82b7d678032e2b4c684189788e175b83d0aa7
+      uri: https://downloads.sourceforge.net/sourceforge/opencore-amr/opencore-amr-${{package.version}}.tar.gz
+
+  - uses: autoconf/configure
+    with:
+      opts: --sysconfdir=/etc --mandir=/usr/share/man --localstatedir=/var
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - uses: strip
+
+subpackages:
+  - name: opencore-amr-dev
+    description: Development files for opencore-amr
+    pipeline:
+      - uses: split/dev
+
```

Comment:
The primary package ships .so files so adding a test with ldd-check is appropriate too.